docs/design-proposals/apiserver/Data-Partition-Design.md

```diff
@@ -0,0 +1,198 @@
+---
+title: API Server Data Partition Design for CloudFabric
```

Comment:
Consider removing "for CloudFabric". It's not necessary And you may need to change documents when this name changes :)

Reply:
changed to Arktos